pages/movies.js

```diff
@@ -0,0 +1,103 @@
+import * as React from "react";
+import {
+  Card,
+  CardContent,
+  CardMedia,
+  Typography,
+  Button,
+  CardActionArea,
+  CardActions,
+  Grid,
+  ThemeProvider,
+  createTheme,
```

Comment:
what is createTheme for?

Reply:
It's a function used to generate a theme: https://mui.com/material-ui/customization/theming/#api

Reply:
And themes / theming are used to sort of make a general layout (or yeah, theme). I wasn't satisfied with the default MUI theme as it uses different fonts, colors and so on compared to our site. But you can, for example, use the sx prop on lower / component and element level, although I mostly played around with theming to get a feel for it.

Reply:
https://mui.com/material-ui/customization/theming/